androidlibrary_lib/src/main/java/org/opendatakit/fragment/AlertDialogFragment.java

```diff
@@ -130,10 +133,10 @@ public boolean dismissWasCalled() {
 
    public boolean createDialogCalled() { return createDialogCalled; }
 
-   private AlertDialog getAlertDialog() {
+   private androidx.appcompat.app.AlertDialog getAlertDialog() {
```

Comment:
Now that the regular `AlertDialog` is not used, you could remove its import and import the AndroidX `AlertDialog`.